src/components/ContextMenu/ContextMenuItem.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import ContextMenu from '.';
+
+function ContextMenuItem({ item, parentRef, isToLeft }) {
+  const [renderMenu, setRenderMenu] = useState(false);
+
+  function handleMouseEnter() {
+    setRenderMenu(true);
+  }
+
+  function handleMouseLeave() {
+    setRenderMenu(false);
+  }
+
+  switch (item.type) {
+    case 'item':
+      return (
+        <StyledItem className={`${item.inactive ? 'inactive' : ''}`}>
+          {item.text}
+        </StyledItem>
+      );
+    case 'menu':
+      return (
+        <StyledItem
+          className={`arrow`}
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
+        >
+          {item.text}
+          {renderMenu && (
+            <ContextMenu
+              items={item.items}
+              parentRef={parentRef}
+              isToLeft={isToLeft}
+            />
+          )}
+        </StyledItem>
+      );
+    case 'separator':
+      return <StyledItem className={`separator`}></StyledItem>;
```

Comment:
Here (and in the error className above) you don't really use anything that justifies the template literal. Please simplify to `className="separator"` etc.

Reply:
Right 🧐  Fixed.